api/api-pipeline-config-v7/src/main/java/com/thoughtworks/go/apiv7/admin/pipelineconfig/PipelineConfigControllerV7.java

```diff
@@ -131,6 +139,10 @@ public String create(Request req, Response res) {
         Username userName = SessionUtils.currentUsername();
         pipelineConfigService.createPipelineConfig(userName, pipelineConfigFromRequest, result, group);
 
+        if(shouldPausePipelineOnCreate(req)){
+            pipelinePauseService.pause(pipelineConfigFromRequest.name().toString(), null, userName);
```

Comment:
Should we add a pause message for this?

Reply:
And default to a sensible message, if one is not specified.

Reply:
@ketan - Are u talking about making changes in <code>PipelinePauseService.pause()</code> method? wherein if caller of that method, does not specify the pause cause, then the sensible default message is used instead. That way pause message is never null in the database after this change. Is that right?